### DISCUSSION TITLE
Add label to 3D splines

### DISCUSSION THREAD

**Discussion by apaaris** [pyvista/pyvista#3235] [Q&A] [answered]
Dear everyone, 

I have a number of 3D splines, which are already colored by their own unique ID/Index. Would it be possible to add a label displaying this ID next to each of the splines as you can do with points?

Something like this:
![image](https://user-images.githubusercontent.com/64897205/187392525-fdcfce76-7c0d-4819-86b5-2215614fafe1.png)
But for splines/3D objects

Thank you a lot

**Comment by tkoyama010** [comment, marked as answer]
You can show the ID of Spline using `add_point_labels`. Please add points and labels to each list.

```python
import numpy as np
import pyvista as pv

theta = np.linspace(-4 * np.pi, 4 * np.pi, 100)
z = np.linspace(-2, 2, 100)
r = z**2 + 1
x = r * np.sin(theta)
y = r * np.cos(theta)
points = np.column_stack((x, y, z))
spline = pv.Spline(points, 1000)
p = pv.Plotter()
p.add_mesh(spline)
# Add center of cells here
points = [spline.center]
# Add labels here
labels = ["1234"]
p.add_point_labels(points=points, labels=labels)
p.show(screenshot="output.png")
```
![output](https://user-images.githubusercontent.com/7513610/187409898-702bc597-7aef-4ddb-b38c-012582354653.png)

**Reply by tkoyama010** [reply]
Add `add_cell_labels` to PyVista in https://github.com/pyvista/pyvista/pull/3237.

**Reply by apaaris** [reply]
This definitely did it for me! Thanks
